cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -793,8 +794,8 @@ class TLMOptions(TypedDict):
     You can set custom values for these arguments regardless of the quality preset specified.
 
     Args:
-        model (str, default = "gpt-4o-mini"): underlying base LLM to use (better models yield better results, faster models yield faster/cheaper results).
-        - Models currently supported include: "gpt-4o-mini", "gpt-4o", "o1-preview", "gpt-3.5-turbo-16k", "gpt-4", "claude-3.5-sonnet", "claude-3-haiku".
+        model ({"gpt-4o-mini", "gpt-4o", "o1-preview", "gpt-3.5-turbo-16k", "gpt-4", "claude-3.5-sonnet", "claude-3-haiku"}, default = "gpt-4o-mini"): Underlying
+        base LLM to use (better models yield better results, faster models yield faster/cheaper results).
         - Additional models supported in beta include: "o1", "o1-mini", "claude-3.5-sonnet-v2", "claude-3.5-haiku", "nova-micro", "nova-lite", "nova-pro".
```

Comment:
I'm not 100% sure how we want to list the beta models here, I personally think listing them in a bullet point is alright but open to suggestions